packages/gafl-webapp-service/src/pages/licence-details/licence-type/update-transaction.js

```diff
@@ -17,6 +20,16 @@ export default async request => {
     permission.licenceType = constants.LICENCE_TYPE['trout-and-coarse']
     permission.numberOfRods = '3'
     permission.licenceLength = '12M'
+    permission.licenceStartTime = null
+    // If the licence start date has previously be chosen as today, for a 12 month
+    // then set start after payment
+    if (
+      moment(permission.licenceStartDate, 'YYYY-MM-DD')
```

Comment:
This is using moment-timezone but you've only imported moment

Reply:
My bad. I didn't know about moment-timezone. Corrected throughout.